.travis.yml

```diff
@@ -1,24 +1,51 @@
 language: php
 
 php:
-  - 5.3.3
   - 5.3
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
+  - nightly
```

Comment:
7.0 and nightly are the same thing currently. So addign both just means extra work on Travis (and the number of concurrent jobs is limited)